### ISSUE TITLE
Chart.js integration

### ISSUE THREAD

**Issue by mikezimm** [mikezimm/TrackMyTime7#20]
## Build basic charts using [chart.js react control](https://sharepoint.github.io/sp-dev-fx-controls-react/controls/ChartControl/)

Basic data object structure required:
``` typescript
  data={{
    labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
    datasets: [{
      label: 'My First dataset',
      data: [65, 59, 80, 81, 56, 55, 40]
    }]
  }}
```

## Source data structure
```javascript

//Basic data datapoint structure
export interface IDataPoint {
  label: string; // "Dec-12", Date or category label
  sort:  number; // 12341324423, milliseconds For sorting element if a date field
  value: number; //20, 
  target: number; //18, Should be in same units as value
  units: string; //  "hours", Maybe make this flexible because of project targets
  targetPercent: number; // 110, 
  backgroundColor?:  'rgba(255, 99, 132, 0.2)'//
  borderColor? 'rgba(255,99,132,1)' //
}

//with different buckets maybe?  Not sure if these should be built at first or on demand
export interface IDataPoints {
  daily: IDataPoint[], //
  timeBuckets: IDataPoint[], //Today, This Week, This Month
  category1: IDataPoint[], //Based on values in category columns
  category2: IDataPoint[], //Based on values in category columns
}

//Probably have 2 different sets of data based on the current user
//Maybe toggle show currentUser by default, then get everyone if you press toggle.
export interface IPersonDataPoints {
  currentUser?: IDataPoints,
  everyone?: IDataPoints,
}

```

## Sort data using array sort:  [www3 array_sort](https://www.w3schools.com/js/js_array_sort.asp)
``` javascript
var cars = [
  {type:"Volvo", year:2016},
  {type:"Saab", year:2001},
  {type:"BMW", year:2010}
];

displayCars();

function myFunction() {
  cars.sort(function(a, b){return a.year - b.year});
  displayCars();
}

```

**Comment by mikezimm** [comment]
Started time entry data structure here:
[2020-02-05-Add-Chart-JS-framework](https://github.com/mikezimm/TrackMyTime7/tree/2020-02-05-Add-Chart-JS-framework)

**Comment by mikezimm** [comment]
Closed basic charting with #37
